content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -153,21 +153,21 @@ image::tutorials/IBMCloud/done.png[]
 ## Verify Jenkins installation
 
 * Go to http://cloud.ibm.com/resources[Resources] in your browser
-* Click on **Clusters**
-* Click on your Cluster
+* Select **Clusters**
+* Select your Cluster
 
 image::tutorials/IBMCloud/resource-select.png[]
 
-* Now you are at you clusters overview, here Click on **Actions** and **Web terminal** from the dropdown menu
+* Now you are at you clusters overview, here Select **Actions** and **Web terminal** from the dropdown menu
```

Comment:
```suggestion
+
Now you are at you clusters overview. 
* From the **Actions** from the dropdown menu, select **Web terminal**.
```

Reply:
```suggestion
* Now you are at you clusters overview.
From the  **Actions**  menu, select **Web terminal** from the dropdown menu.
```

Reply:
Hi @kellie-freeman, would my suggestion work in this case as well? I want to make sure that things are described as clearly as possible. Thanks!